src/base/porting.c

```diff
@@ -1495,7 +1500,7 @@ cub_vsnprintf (char *buffer, size_t count, const char *format, va_list argptr)
       buffer[count - 1] = 0;
 
       free (cp);
-      return (int) count;
+      return (int) len;
```

Comment:
```suggestion
      return (int) (len - 1);
```
man vsnprintf 를 참고하면
If the output was truncated due to this limit then the return value is the number of characters (**excluding  the  terminating null byte**) which would have been written to the final string if enough space had been available.

Reply:
1492라인에서 새로 얻는 `len`은 충분한 버퍼가 준비된 상태에서 수행되는 것입니다.
현재 함수의 동작 결과가 리눅스와도 일치합니다.

Reply:
아. len 도 음수 이군요. 알겠습니다.

Reply:
아. 새로 얻은 len 이군요. 잘못 봤습니다.